cloud-init.yml

```diff
@@ -1,227 +1,227 @@
-#cloud-config
```

Comment:
Same thing. It's really hard to review this if the whole file is completely replaced.